dev/com.ibm.ws.security.acme/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -70,6 +70,8 @@
    <AD id="trustStore"           name="%trustStore"         description="%trustStore.desc"         required="false" type="String" />
    <AD id="trustStorePassword"   name="%trustStorePassword" description="%trustStorePassword.desc" required="false" type="String" ibm:type="password" />
    <AD id="trustStoreType"       name="%trustStoreType"     description="%trustStoreType.desc"     required="false" type="String" />
+   <AD id="httpConnectTimeout"   name="internal" description="internal use only" type="Integer"  required="false" default="10000" min="0" />
```

Comment:
Shouldn't these be `ibm:type="duration"`?

Reply:
Changed to duration